src/components/ButtonGroup/__tests__/buttonGroup-snapshot-tests.jest.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import ButtonGroup from "../ButtonGroup";
+
+const option = [
+  { value: 1, text: "Alpha" },
+  { value: 2, text: "Beta" },
+  { value: 3, text: "Gamma" },
+  { value: 4, text: "Delta" }
+];
+
+describe("ButtonGroup renders correctly", () => {
+  it("with value", () => {
+    const tree = renderer.create(<ButtonGroup value="1" options={option} />).toJSON();
```

Comment:
Since the value of the first number is numeric (1 - number), therefore if we will set the ButtonGroup.value = "1" (string), no button will be selected and the test actually does not check what it intend to check.
please change the value prop to 1 (number)